src/shaders/primitive-id-textured-gl410.frag

```diff
@@ -1,8 +1,10 @@
 in mediump vec3 v_color;
 flat in mediump uint v_objectId;
+in int gl_PrimitiveID;
```

Comment:
We need an #ifdef in the shared or two separate shaders with an #ifdef choosing which shader to use.